packages/@okta/vuepress-site/docs/reference/api/rulesets/index.md

```diff
@@ -0,0 +1,979 @@
+
+Rule Sets also introduce the concept of `Mappings`, which represent the governance relationship of a policy over a given resource.
+
+Currently Okta allows a single Resource Mapping for each Rule Set, but the API will naturally support multiple mappings in the future.
+
+> If you want to change the Rule Set governing a particular resource, you must delete the old mapping and create a new one.
+
+## Limits
+
+Okta limits the number of Rule Sets to 100 per [type](#rule-set-types).
+
+## Getting Started
+
+The Policy API supports the following **rule set operations**:
+
+* Get all rule sets of a specific type
+* Create, read, update, and delete a rule set
+* Activate and deactivate a rule set
+
+The Policy API supports the following **rule operations**:
+
+* Get all rules for a rule set
+* Create, read, update, and delete a rule for a rule set
+* Activate and deactivate a rule
+
+The Policy API supports the following **resource mapping operations**:
+
+* Get all resource mappings for a rule set
+* Create, read, update, and delete a resource mapping for a rule set
+
+## Rule Set Operations
+
+### Get a Rule Set
+
+<ApiOperation method="get" url="/api/v1/policies/${ruleSetId}" />
+
+##### Request Parameters
+
+The rule set ID described in the [RuleSet object](#rule-set-object) is required.
+
+##### Request Example
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/policies/{rulesetId}"
+```
+
+##### Response Example
+
+HTTP 200:
+[RuleSet object](#rule-set-object)
+
+### Get All Rule Sets by Type
+
+<ApiOperation method="get" url="/api/v1/policies?type=${type}" />
+
+##### Request Parameters
+
+The rule set type described in the [RuleSet object](#rule-set-object) is required.
+
+##### Request Example
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/policies?type={type}"
+```
+
+##### Response Types
+
+HTTP 200:
+[RuleSet object](#rule-set-object)
+
+### Delete Rule Set
+
+<ApiOperation method="delete" url="/api/v1/policies/${rulesetId}" />
+
+##### Request Parameters
+
+The rule set ID described in the [RuleSet object](#rule-set-object) is required.
+
+##### Request Example
+
+```bash
+curl -v -X DELETE \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/policies/{rulesetId}"
+```
+
+##### Response Types
+
+HTTP 204:
+*No Content*
+
+### Update a Rule Set
+
+<ApiOperation method="put" url="/api/v1/policies/${rulesetId}" />
+
+##### Request Parameters
+
+The rule set ID described in the [RuleSet object](#rule-set-object) is required.
+
+##### Request Example
```

Comment:
Which properties can or can't be updated via API?

Reply:
You can only update:
* status (if it's not the default)
* name

Reply:
Not `default`?

Reply:
@jakubvul-okta did you need more clarification on this?